app/models/case_log.rb

```diff
@@ -157,12 +157,12 @@ def applicable_income_range
 private
 
   RENT_TYPE_MAPPING = {
```

Comment:
This might be better living in the `constants` module now since it seems like it's doing the same job as all the others (mapping values)?

Reply:
Ohh, good shout!